weather_dl_v2/fastapi-server/config_processing/pipeline.py

```diff
@@ -63,6 +63,9 @@ async def start_processing_config(config_file, licenses, force_download):
             config_name, "Partitioning completed."
         )
         await queue_handler._update_queues_on_start_download(config_name, licenses)
+        if priority:
+            for license_id in licenses:
+                await queue_handler._update_config_priority_in_license(license_id, config_name, priority)
```

Comment:
Discussed offline, let's try to do this one call; instead of making 2 DB calls.

Reply:
done